### ISSUE TITLE
Operator precedence and units

### ISSUE THREAD

**Issue by clinssen** [nest/nestml#1128]
The NESTML string "0 ms**-1" can be read as either "0 (ms**-1)", a rate of change of 0 per millisecond, or as "(0 ms)**-1", a division by zero error.

(To obtain the former, one can write "0 * ms**-1".)

Possibly, operator precedence rules should be changed such that the former interpretation is the default.

**Comment by tomtetzlaff** [comment]
I agree. By default, I would always intepret the empty space between "0" and "ms" in "0 ms" as a product "0*ms", similarly to how things are in real life. With this and with the usual operator precedences, "0 ms**-1" would mean "0*(ms**-1)".
